build.cmd

```diff
@@ -18,6 +18,12 @@ if %errorlevel% equ 0 (
   set outloop=true
 )
 
+echo %* | findstr /i /C:"FloatingTestRuntimeDependencies=true"  1>nul
+if %errorlevel% equ 0 (
+  set TestFixedRuntimeProjectJson=%~dp0src\Common\test-runtime\LatestDependencies\project.json
+  set defaultBuildConfig=/p:Configuration=Windows_NT_Debug
```

Comment:
I think scenario 3 still requires us to be able to build WCF packages that depend on the latest versions.  And then we can do the manual E2E verification using those.  It's not automated, but it would still require making sure build.cmd built the right packages with the new dependencies.